src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/X86Base.PlatformNotSupported.cs

```diff
@@ -109,6 +128,26 @@ internal X64() { }
         [Experimental(Experimentals.X86BaseDivRemDiagId, UrlFormat = Experimentals.SharedUrlFormat)]
         public static (nint Quotient, nint Remainder) DivRem(nuint lower, nint upper, nint divisor) { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        ///   <para>  MUL reg/m32</para>
+        ///   <para>  MULX reg reg reg/m32 (if BMI2 is supported)</para>
+        /// </summary>
+        internal static (uint Lower, uint Upper) BigMul(uint left, uint right) { throw new PlatformNotSupportedException(); }
+
+        /// <summary>
+        ///   <para>  IMUL reg/m32</para>
+        /// </summary>
+        internal static (int Lower, int Upper) BigMul(int left, int right) { throw new PlatformNotSupportedException(); }
+
+        /// <summary>
+        ///   <para>  MUL reg/m</para>
+        ///   <para>  MULX reg reg reg/m (if BMI2 is supported)</para>
+        /// </summary>
+        internal static (nuint Lower, nuint Upper) BigMul(nuint left, nuint right) { throw new PlatformNotSupportedException(); }
+
+        /// <summary>  IMUL reg/m</summary>
+        internal static (nint Lower, nint Upper) BigMul(nint left, nint right) { throw new PlatformNotSupportedException(); }
```

Comment:
The nint and 32 bit version never gets called, should I remove them or keep them for the symmetry ? 

I do no longer think it makes much sense to make this methods public (#58263) since you can just call existing BigMul methods, and the nint version was solved by a simple *if * in #114731